decidim-admin/app/packs/stylesheets/decidim/admin/_redesigned_sidebar-menu.scss

```diff
@@ -0,0 +1,61 @@
+.sidebar-menu {
+  @apply flex-none w-full md:w-fit max-w-full md:max-w-[10%];
```

Comment:
Using those arbitrary values just feel wrong. They smell like a Tailwind antippatern. Also with 10% it seems like its too crowded, at least in some viewports. I think it looks a bit better with this:
```suggestion
  @apply flex-none w-full md:w-fit max-w-full md:max-w-lg;
```

Reply:
That LG may not be the best option.  ( this means a max width of 512px) 
![image](https://github.com/decidim/decidim/assets/105683/3e4de285-6a16-4c51-b218-f2afd574432e)

I would opt in for XS ( max width of 320px) 
![image](https://github.com/decidim/decidim/assets/105683/f878389d-a783-4412-8744-cca5c9ca878f)

Reply:
I agree, lets go with xs !